### PR TITLE
Smbghost scanning module

### PR DESCRIPTION
**Pull request by r4vanan**
Module to Scan for the SMBGhost (CVE-2020-0796) Vulnerability with the help of @ly4k script
image

### CODE REVIEW

**Comment by NeffIsBack** [comment]
Thanks for the PR! Cool one 🔥

File: nxc/modules/smbghost.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    self.context.log.debug(f"{target_ip} Not vulnerable.")
+                    return False
+        except Exception as e:
+            self.context.log.fail(f"Error while connecting to host: {e}")
+            return False
+
+def fail(msg):
+    nxc_logger.debug(msg)
+    nxc_logger.fail("This might have been caused by invalid arguments or network issues.")
+    sys.exit(2)
```
Comment (NeffIsBack) [inline nxc/modules/smbghost.py:65]: Looks like this isn't used and could be removed.

**Review by NeffIsBack — CHANGES_REQUESTED** [review]
How did you test it? Is there any VM online available what i could download for testing?

File: nxc/modules/smbghost.py (hunk 1 above)
Comment (r4vanan) [reply nxc/modules/smbghost.py]: that code is removed i will share vulnerable vm and share it to you. i test with vulnerable vm

Comment (NeffIsBack) [reply nxc/modules/smbghost.py]: Thanks a lot!

**Review by NeffIsBack — APPROVED** [review]
LGTM
image

**Review by NeffIsBack — APPROVED** [review]
LGTM

**Pull request merged**